portal/src/main/webapp/WEB-INF/jsp/cross_cancer_results.jsp

```diff
@@ -80,12 +80,15 @@
 
 <jsp:include page="global/header.jsp" flush="true"/>
 
+<script type="text/javascript" src="js/src/modifyQuery.js?<%=GlobalProperties.getAppVersion()%>"></script>
 
 <script>
 window.appVersion = '<%=GlobalProperties.getAppVersion()%>';
     
 window.historyType = 'memory';
 
+window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
```

Comment:
probably what we should do is put all these things in include.  That way, we don't get bit when we replace these pages.   But I'll accept this PR and then we can follow up with that.